infra/base-images/base-builder/compile

```diff
@@ -65,10 +65,18 @@ echo "CXXFLAGS=$CXXFLAGS"
 echo "---------------------------------------------------------------"
 
 BUILD_CMD="bash -eux $SRC/build.sh"
+POST_BUILD_CMD="cp -r $SRC $WORK $OUT"
```

Comment:
nit: add a comment here about why we do this, and rename to something more meaningful, like "COPY_SOURCES_CMD"

Reply:
Done.